beta-src/src/components/ui/WDCommandButtons.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import { Stack } from "@mui/material";
+import WDCheckmarkIcon from "../svgr-components/WDCheckmarkIcon";
+import WDButton from "./WDButton";
+import VoteType from "../../types/Vote";
+import Vote from "../../enums/Vote";
+
+interface voteProps {
+  voteState: VoteType;
+  toggleVote: (vote: Vote) => void;
+}
+
+const WDCommandButtons: React.FC<voteProps> = function ({
+  voteState,
+  toggleVote,
+}): React.ReactElement {
+  const { draw, pause, cancel } = voteState;
+
+  return (
+    <Stack direction="row" spacing={2} alignItems="center">
+      <WDButton
```

Comment:
These three buttons seem pretty similar. Maybe we can have a function to return corresponding button based on values passed.

Reply:
seems we have the WDButton component already, once its approved I will just make a function to convert the VoteType to array and do a map then